gcloud/bigquery/client.py

```diff
@@ -96,3 +97,20 @@ def dataset(self, name):
         :returns: a new ``Dataset`` instance
         """
         return Dataset(name, client=self)
+
+    def load_from_storage(self, name, destination, *source_uris):
```

Comment:
I don't think `job` is crucial, just some indication of what is being loaded. I think `async` would be just fine. `load_table_from_storage` probably also fine.